adobe_vipm/adobe/mixins/transfer.py

```diff
@@ -113,8 +146,15 @@ def preview_reseller_change(
                 "action": "PREVIEW",
                 "approvalCode": change_code,
                 "resellerId": reseller.id,
-                "requestedBy": admin_email
+                "requestedBy": admin_email,
             },
+            timeout=self._TIMEOUT,
         )
         response.raise_for_status()
         return response.json()
+
+    def _do_not_make_return_params(self):
```

Comment:
It's strange that mypy doesn't report anything about the typing. Do we have the correct config?

```suggestion
    def _do_not_make_return_params(self) -> dict[str, str]:
```

Reply:
Ah! I've just realized we don't have mypy in this repo. 😢